experimental/dds/tree2/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -986,6 +989,8 @@ export function splitMark<T, TMark extends Mark<T>>(
 			}
 			return [mark1, mark2];
 		}
+		case "Placeholder":
+			fail("TODO");
```

Comment:
Is this a "TODO before merging" or a "TODO after merging"?
I'm guessing it's the latter, and you don't want to both with doing it now because you're about to overhaul how the splitting works. Is that right?

Reply:
Yes. It's also not easy to write a test which exercises this behavior and I don't think it will get hit in real use cases in the near term.